spec/update_spec.rb

```diff
@@ -593,237 +590,152 @@ def update(revisions, user_type=:editor)
-        filename: '::blank',
-        original_filename: '::blank'
-      }.to_json)
+      # we have created some new records
+      expect(Labors::Prize.count).to eq(2)
 
-      expect(last_response.status).to eq(200)
-
-      # and we do not get an upload url for Metis
-      expect(json_document(:monster, 'Nemean Lion')[:selfie]).to eq({
-        path: '::blank'})
+      # the prizes are linked to the labor
+      labor.refresh
+      expect(labor.prize.count).to eq(2)
 
-      # Make sure the Metis copy endpoint was not called
-      expect(WebMock).not_to have_requested(:post, "https://metis.test/labors/files/copy").
-      with(query: hash_including({
-        "X-Etna-Headers": "revisions"
-      }))
+      # the updated record is returned
+      expect(last_response.status).to eq(200)
+      expect(json_document(:labor, 'The Golden Apples of the Hesperides')[:prize]).to match_array(Labors::Prize.select_map(:id))
     end
 
-    it 'returns a temporary Metis path when using ::temp' do
-      lion = create(:monster, name: 'Nemean Lion', species: 'lion')
-
+    it 'appends to an existing table' do
+      labor = create(:labor, name: 'The Golden Apples of the Hesperides')
+      apples = create_list(:prize, 3, @apple_of_discord.merge(labor: labor))
       update(
-        monster: {
-          'Nemean Lion' => {
-            selfie: {
-              path: '::temp'
-            }
-          }
+        'prize' => {
+          '::temp1' => @apple_of_joy.merge(labor: 'The Golden Apples of the Hesperides'),
+          '::temp2' => @apple_of_joy.merge(labor: 'The Golden Apples of the Hesperides')
         }
       )
 
-      # the field is updated
-      lion.refresh
-      expect(lion.selfie).to eq(nil)
-
-      expect(last_response.status).to eq(200)
+      # we have created some new records
+      expect(Labors::Prize.count).to eq(5)
 
-      # but we do get an upload url for Metis
-      upload_url = json_document(:monster, 'Nemean Lion')[:selfie][:path]
-      expect(upload_url.
-        start_with?('https://metis.test/labors/upload/magma/tmp/')).to eq(true)
-      expect(upload_url.
-        include?('X-Etna-Signature=')).to eq(true)
+      # the prizes are linked to the labor
+      labor.refresh
+      expect(labor.prize.map(&:name)).to match_array([ 'apple of joy' ] * 2 + [ 'apple of discord' ] * 3)
 
-      # Make sure the Metis copy endpoint was not called
-      expect(WebMock).not_to have_requested(:post, "https://metis.test/labors/files/copy").
-      with(query: hash_including({
-        "X-Etna-Headers": "revisions"
-      }))
+      # the updated record is returned
+      expect(last_response.status).to eq(200)
+      expect(json_body[:models][:prize][:documents].values.map{|p|p[:name]}).to eq(['apple of joy']*2)
     end
 
-    it 'links an image from metis' do
-      Timecop.freeze(DateTime.new(500))
-      lion = create(:monster, name: 'Nemean Lion', species: 'lion')
+    xit 'replaces an existing table' do
```

Comment:
Is this test supposed to be skipped?

Reply:
Yes, both of these should be skipped. I would like this behavior to work but didn't have time to implement it for now, so I opted to leave a skipped test in place as a reminder.